plenum/test/view_change/test_primary_send_incorrect_pp.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+
+from plenum.common.messages.node_messages import OldViewPrePrepareReply, PrePrepare
+from plenum.test.delayers import lsDelay, msg_rep_delay, ppDelay
+from plenum.test.node_request.helper import sdk_ensure_pool_functional
+from plenum.test.stasher import delay_rules, delay_rules_without_processing
+from plenum.test.view_change.helper import ensure_all_nodes_have_same_data, \
+    ensure_view_change, add_new_node
+from plenum.common.constants import DOMAIN_LEDGER_ID, LedgerState, POOL_LEDGER_ID, LEDGER_STATUS, PREPREPARE
+from plenum.test.helper import sdk_send_random_and_check, waitForViewChange, assertExp, sdk_send_random_request, \
+    sdk_get_and_check_replies
+
+from stp_core.common.log import getlogger
+from stp_core.loop.eventually import eventually
+from plenum.test.node_catchup.helper import check_ledger_state, \
+    waitNodeDataEquality
+from plenum.common.util import randomString
+from plenum.test.test_node import checkNodesConnected, ensureElectionsDone
+from plenum.test.pool_transactions.helper import sdk_add_new_steward_and_node, sdk_pool_refresh
+from plenum.test import waits
+from plenum.common.startable import Mode
+
+logger = getlogger()
+
+
+def test_primary_send_incorrect_pp(looper, txnPoolNodeSet, tconf,
+                                   allPluginsPath, sdk_pool_handle,
+                                   sdk_wallet_steward,
+                                   monkeypatch):
+    """
+    Test steps:
+    Delay PrePrepares on `slow_node` (without processing)
+    Delay receiving of OldViewPrePrepareRequest on all nodes but `malicious_node`
+    Patch sending for OldViewPrePrepareReply on the `malicious_node` to send an invalid PrePrepare
```

Comment:
Looks like invalid description. We are patching `PrePrepare`, not `OldViewPrePrepareReply `, right?